gnocchiclient/v1/archive_policy_cli.py

```diff
@@ -54,6 +54,8 @@ def archive_policy_definition(string):
     defs = {}
     for part in parts:
         attr, __, value = part.partition(":")
+        attr = attr.replace(' ', '')
+        value = value.replace(' ', '')
```

Comment:
i think you can put `parts = string.replace(' ', '').split(",")` in line 53 so you just need  to run replace once and be done.

Reply:
Thank you for your suggestion. But I found the travis CI build failed. I analyzed the test code and can be run, so I don't know the cause of the failure.